meson.build

```diff
@@ -1460,8 +1460,12 @@ videotoolbox_gl = get_option('videotoolbox-gl').require(
     error_message: 'gl-cocoa nor ios-gl could be found!',
 )
 features += {'videotoolbox-gl': videotoolbox_gl.allowed()}
+features += {'videotoolbox-pl': videotoolbox_gl.allowed() and libplacebo.found()}
```

Comment:
Haven't looked at the changes in detail yet, but libplacebo is a hard requirement now so you don't need that check.

Reply:
Okay so looking at this check again I realize that it incurs both a GL dependency and also would pass if `ios-gl` exists but not `gl-cocoa`. I don't use this stuff so I don't really know but is that intended?

Reply:
This was basically just to have _something_ to check for that'd cover both iOS and macOS; I switched it to check for CoreVideo.